docs/user-guide/diff-strategies.md

```diff
@@ -17,16 +17,14 @@ Argo CD currently has 3 different strategies to calculate diffs:
 
 ## Structured-Merge Diff
 
-!!! warning "Beta Feature (Since v2.5.0)"
-    This feature is in the [Beta][1] stage. It is generally considered stable, but there may be unhandled edge cases.
+!!! warning "Feature Discontinued"
```

Comment:
Is this true?

Reply:
It was the last sentence of the paragraph. I have just moved it to a more prominent place, as IMO it makes no sense to warn the feature is in beta _and_ that it will be removed. In the light of the latter, the former is insignificant.